### DISCUSSION TITLE
Profile page domain

### DISCUSSION THREAD

**Discussion by praddasg** [community/community#23912] [New to GitHub]
<p>I created repo as <code>x.github.io</code>, which I can access fine. I created another repo <code>y.github.io</code> in the same way but I am getting HTTP 404.</p>
<p>Following this <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site" rel="noopener nofollow ugc">doc</a>, published the page but now the webpage is <code>x.github.io/y.github.io</code>. Any idea why like this not <code>y.github.io</code>?</p>
<p>Also couple more questions</p>
<ol>
<li>Why I had to publish the 2nd repo and the first one was not required?</li>
<li>For both the repos, I did not create any branch but <code>x.github.io</code> have a master branch and <code>y.github.io</code> has main branch. Any idea?</li>
</ol>

**Comment by praddasg** [comment]
<p>Ok I guess I found the answer to my first question from <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites" rel="noopener nofollow ugc">here</a> i.e. can only create one site with username. In this case <code>x</code> was indeed my user name. For rest of the site, have to create custom domain.</p>
<p>I guess that also answer my next question <code>Why I had to publish the 2nd repo and the first one was not required?</code>, since this require to define a custom domain name I am guessing. Would love to get a confirmation</p>
<p>My last question still remains unanswered though</p>

**Comment by tajmone** [comment, marked as answer]
<p>GitHub Pages allows you to have one website for you personal account, and one website for each repository. URLs for each website are determined by  you username and repository name, you can’t change that unless you have your custom domain which you connect to a GH Pages website, via the configuration of the specific repository.</p>
